cmd/influx/task.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"github.com/influxdata/influx-cli/v2/internal/cmd/task"
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli/v2"
+)
+
+func newTaskCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "task",
+		Usage: "User management commands",
+		Subcommands: []*cli.Command{
+			//newTaskLogCmd(),
+			//newTaskRunCmd(),
+			newTaskCreateCmd(),
+			//newTaskDeleteCmd(),
+			//newTaskFindCmd(),
+			//newTaskUpdateCmd(),
+			//newTaskRetryFailedCmd(),
+		},
+	}
+}
+
+func newTaskCreateCmd() *cli.Command {
+	var params task.CreateParams
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, &cli.StringFlag{
+		Name:        "file",
+		Usage:       "Path to Flux script file",
+		Aliases:     []string{"f"},
+		TakesFile:   true,
+		Destination: new(string),
```

Comment:
Should be ok to omit `Destination` if you're not trying to write into a local var